config/vufind/EDS.ini

```diff
@@ -92,6 +92,16 @@ load_results_with_js = true
 ; full                   Full pagination alike to the one at the bottom of results
 ;top_paginator = simple
 
+; Minimum value for date range sliders in search forms (default is 1400). Value
+; must be a number or a string that PHP's strtotime can parse. Only the year part
+; is used.
+;date_range_slider_min = "1400"
```

Comment:
We might also want to make Summon.ini match, since there's a date range facet there as well. (I realize that we can't test it, but I think I can recruit some help with that if we need it).

Reply:
Done. I can't test it either, but at least the slider reflects the settings now. :)